02/app.js

```diff
@@ -1,7 +1,26 @@
+// /* rozwiązanie z pętlą for */
+const x = parseInt(prompt('podaj liczbę do mnożenia'))
+for (let i = 1; i < 10; i++) {
+	const element = x * i
+	if (x < 1 && x >= 10) {
+		console.log('nie poprawna liczba')
+	} else {
+		console.log(`${x} x ${i} =${element}`)
+	}
```

Comment:
Zwróć uwagę, że ten if sprawdza wieloktronie to samo.
To sprawdzenie powinno być przed pętlą `for` czyli:
1. Najpierw pobiera dane od użytkoniwka
2. Sprawdzam ich poprawność
3. Wykonuję pętlę.
4. Wyświetlam wynik dla danych z pętli

U Ciebie zmieniani ona jest kolejność 2 i 3.